src/pages/task/actionButtons/relevantCommits/index.tsx

```diff
@@ -85,6 +86,33 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
     },
   });
   const lastPassingTask = getTaskFromMainlineCommitsQuery(lastPassingTaskData);
+  const passingOrderNumber = lastPassingTask?.order;
+
+  // The breaking commit is the first failing commit after the last passing commit.
+  // The skip order number should be the last passing commit's order number + 1.
+  // We use + 2 because internally the query does a less than comparison.
+  const { data: breakingTaskData, loading: breakingLoading } = useQuery<
+    LastMainlineCommitQuery,
+    LastMainlineCommitQueryVariables
+  >(LAST_MAINLINE_COMMIT, {
+    skip:
+      !parentTask ||
+      !lastPassingTask ||
+      status === undefined ||
+      status === TaskStatus.Succeeded,
```

Comment:
Hm, I don't think this would eliminate it since we should check for `!isFailedTaskStatus` which `undefined` would return `true`, then the `!` would make it `false` and not skip for undefined status.
I think I could keep the undefined check or switch it to do `status ?? TaskSucceeded` (not sure how to reference task success status but however it is)- but that may be confusing. What do you think?

Reply:
I feel like `isFailedTaskStatus(status)` should return `false`  for an `undefined` status, and then the `!isFailedTaskStatus(status)` should make it `true` to skip. Is that not the case for the function? 👀

Reply:
You are right, not sure what I was thinking when I wrote the above... Thank you for being thorough!

Reply:
You are right, not sure what I was thinking when I wrote the above... Thank you for being thorough!